doc/cephfs/snap-schedule.rst

```diff
@@ -101,10 +101,10 @@ path.
 
 Examples::
 
-  ceph fs snap-schedule status /
-  ceph fs snap-schedule status /foo/bar --format=json
-  ceph fs snap-schedule list /
-  ceph fs snap-schedule list / --recursive=true # list all schedules in the tree
+  ceph fs snap-schedule status / --fs docs
```

Comment:
for someone reading this doc for the first time might get confused whether about `docs` fs being mandatory, this can be made self explanatory with like `myfs` or `your_fs_name`?

Reply:
updated the docs to reflect the fs name hint